src/command_modules/azure-cli-keyvault/HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+2.3.0
++++++
+* BREAKING CHANGE: updated keyvault create command to fail by default if the vault alread exists.  The previous behavior can still be achieved by adding the --force parameter.  See issue https://github.com/Azure/azure-cli/issues/7608.
```

Comment:
This breaks the idempotency of `keyvault create`. We are aware of this ask via https://github.com/Azure/azure-cli/issues/7613 and want to ensure we address is consistently across the CLI. I would ask that you remove this breaking change for now.

Reply:
In that case I'm going to close this PR and open a new one with just the one fix.